Eplant/UI/LeftNav/GeneSearch/SearchBar.tsx

```diff
@@ -32,6 +32,15 @@ export default function SearchBar(props: {
   const updateOptions = React.useRef<(text: string) => void>()
   const theme = useTheme()
 
+  function handleChange(input: string[]){
+    if(input.length > 0){
+      // If input is a comma seperated list, will split into seperate genes 
```

Comment:
This case is probably impossible, but if the input looks like `["gene1,gene2", "gene3,gene4"]` it won't be split correctly. Should we just assume that it will never happen?

Reply:
It happens when you have one gene in the list and you try to delete it. When this happens an empty array is passed to the change handler and it throws an error when trying to splice the list. I hadn't considered the above case but it seems like it should work fine right? 

I did just notice that typing or pasting a quotation mark crashes the application (this happens on the live version as well)

Reply:
Yeah I think it should be fine. Good catch about the quotation mark